### DISCUSSION TITLE
Ostrzeżenia tłumaczone na żywo wtyczką "Google"

### DISCUSSION THREAD

**Discussion by xMaap** [FiltersHeroes/KAD#1946] [Pomysły]
### Link bezpośredni do strony zawierającej oszustwo

```markdown
btcforex.uk
apextradings.com
```

### Dodatkowe informacje mogące mieć znaczenie

https://forexrev.pl/ostrzezenia-26-10-igfb-alpha-capital-trade-swiss24/

**Comment by hawkeye116477** [comment]
`btcforex.uk` nie mają polskiej strony, a Tłumacz wujaszka się nie liczy.
`apextradings.com` podobnie.

**Comment by xMaap** [comment]
@hawkeye116477 to coś się zmieniło? Kiedyś @krystian3w pisał, że z tłumaczem na stronie też przechodzi.

**Comment by ClusterByte-x** [comment]
To jaki ostatecznie jest konsensus? 🤔

Z tego co pamietam, strony z tłumacza były klasyfikowane do filtra.

**Reply by krystian3w** [reply]
Na pewno zacznie odpadać translacja co się nie zaczęła po załadowaniu strony. Niby każdy może robić ciastko: `googtrans:/auto/pl` czy nawet umiejętnie ukrywać interfejs Google swoim plikiem CSS.

Jeszcze ktoś musiałby dziennie sprawdzać język w CERT/LWS:

https://github.com/FiltersHeroes/KAD/commit/09fc319d10500bf7b31345dcd5427c70320bd948#diff-269381da98ff06dde11226ef06019e35651b7f1ddf78a517bb878a5dd5eaf038R13614 - to nie ma polonizacji ale CERT poparł bana.

**Reply by hawkeye116477** [reply]
> to nie ma polonizacji ale CERT poparł bana.

Cóż, u nas Steam na pewno jest popularny, w przypadku sporej liczby osób tylko Steam się liczy, dlatego też zapewne dodali...

Myślę, że raczej po CERT nie będziemy poprawiać. Natomiast sami nie będziemy dodawać stron, w przypadku których tłumaczenie nie zaczęło się po załadowaniu strony.
